### ISSUE TITLE
White Smoke and Full Metal Body need test files.

### ISSUE THREAD

**Issue by AlexOn1ine** [rh-hideout/pokeemerald-expansion#4834]
### Description

White Smoke and Full Metal Body need test files with a comment that point to Clear Body

### Version

1.8.4 (Latest release)

**Comment by Pawkkie** [comment]
Just to clarify, you want these to be fully blank files, named normally, where their entire contents is:

```
// These tests are covered in the tests for ABILITY_CLEAR_BODY, and can be found in clear_body.c
```
I'm happy to PR those, I deleted them because I based the style off of Dazzling and its equivalent abilities (Armor Tail and Queenly Majesty) didn't have empty files with comments. I'm happy to add those two also! Just want to make sure this comment is all you'd want so the review process is straightforward :)

**Comment by pkLucid** [comment]
> Just to clarify, you want these to be fully blank files, named normally, where their entire contents is:
> 
> ```
> // These tests are covered in the tests for ABILITY_CLEAR_BODY, and can be found in clear_body.c
> ```
> 
> I'm happy to PR those, I deleted them because I based the style off of Dazzling and its equivalent abilities (Armor Tail and Queenly Majesty) didn't have empty files with comments. I'm happy to add those two also! Just want to make sure this comment is all you'd want so the review process is straightforward :)

Correct. Best would be to use one existing format:
```
#include "global.h"
#include "test/battle.h"

// Tests for Armor Tail are handled in test/battle/ability/dazzling.c
```

**Comment by AlexOn1ine** [comment]
> Just to clarify, you want these to be fully blank files, named normally, where their entire contents is:
> 
> ```
> // These tests are covered in the tests for ABILITY_CLEAR_BODY, and can be found in clear_body.c
> ```
> 
> I'm happy to PR those, I deleted them because I based the style off of Dazzling and its equivalent abilities (Armor Tail and Queenly Majesty) didn't have empty files with comments. I'm happy to add those two also! Just want to make sure this comment is all you'd want so the review process is straightforward :)

Correct!

Best to use an existing format from the repo
```
#include "global.h"
#include "test/battle.h"

// Tests for Armor Tail are handled in test/battle/ability/dazzling.c
```

**Comment by Pawkkie** [comment]
Must've had the wrong branch checked out when I looked for those, didn't see they already had placeholders. PR'd :D

**Comment by Bassoonian** [comment]
Fixed by #4837
